jobs/integration/utils.py

```diff
@@ -183,13 +183,14 @@ async def scp_to(local_path, unit, remote_path, controller_name, connection_name
 async def retry_async_with_timeout(
     func,
     args,
+    kwds=None,
```

Comment:
the async method in `func` will be called with `*args` -- so let's add `kwds` here so that async can have keyword-arguments as well.